src/SourceBuild/tarball/content/ArcadeOverrides/AfterSourceBuild.proj

```diff
@@ -72,7 +72,8 @@
       OutputBaselineFile="$(SourceBuildSelfPrebuiltReportDir)generated-new-baseline.xml"
       OutputReportFile="$(SourceBuildSelfPrebuiltReportDir)baseline-comparison.xml"
       AllowTestProjectUsage="$(AllowTestProjectUsage)"
-      ContinueOnError="$(ContinueOnPrebuiltBaselineError)" />
+      ContinueOnError="$(ContinueOnPrebuiltBaselineError)"
+      Condition="'$(DotNetBuildFromSourceFlavor)' != 'Product'" />
```

Comment:
Do we want to completely disable the prebuilt check here or just always turn on `ContinueOnPrebuiltBaselineError` when we're not in an end-to-end build?

Reply:
Good question.  I was thinking this is a source of distractions/confusion because of the difference in the PVP flow between a repo build and a full source-build.  This is the case in main right now where the source-build-externals baseline is failing because it is picking up newer/different package versions.  What value do you see in having this check in the product source-build?

Reply:
I was thinking that having the generation of the new baseline at least had some value since it's one less full cycle to fix the baseline later and at one point I think we were saving them off for comparison purposes.  I guess it's not that much time saved though and it's not great to have constant warnings that are always ignored so I'm fine with leaving as-is.

Reply:
I am open to refactoring in the future if we learn something interesting.  The problem with updating the baseline in the context of the full build is that it more than likely is not applicable in the repo context given the differences in the pvp flow.